linkerd/examples/socket-options.yaml

```diff
@@ -0,0 +1,16 @@
+namers:
+  - kind: io.l5d.fs
```

Comment:
I think this indented farther than we typically do for lists (see other configs in this directory).  The fact that both are apparently valid yaml is crazy :exploding_head: !  We should be consistent.